index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div class="navbar">
+      <a class="active" href="#home">Home</a>
+      <a href="#portfolio">Portfolio Link</a>
+      <a href="#contact">Contact Link</a>
+      <a href="#about us">About Us</a>
+    </div>
+
+    <div class="title">
+      <h1>Bilal Gujjar Portfolio</h1>
+      <h3>Web Develpment Project</h3>
+    </div>
```

Comment:
Kindly change the main container div tag to a header, that makes for an organized code.